client/src/Forms/RequestForm.js

```diff
@@ -1,11 +1,85 @@
-import React from 'react';
+import React, {Component} from 'react';
+import Form from './FormComponent';
+import {theme, ThemeProvider} from "../Libraries/ReactToolboxLibrary";
 
-const RequestForm = () => {
+/*const RequestForm = () => {
     return(
       <div style={{"display":"flex","width":"100%","justifyContent":"center"}}>
       <iframe src="https://docs.google.com/forms/d/e/1FAIpQLSeJjF6QREqxIZDOc5t6QmXtAbjTxTVjaMZoUOJU1TX-gyk9bw/viewform?embedded=true" width="80%" height="562px" frameborder="0" marginheight="0" marginwidth="0">Loading...</iframe>
       </div>
     );
 }
 
-export default RequestForm;
\ No newline at end of file
+export default RequestForm;*/
+
+export default class RequestForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {Term:'', showErrorTerm: false, errorStyleTerm: 'formInput', stateBar: false, snackbarMessage: ''};
+  }
+
+  createContent = () => {
+    return (<div>
+      <p>I'm collecting informal definitions to help people understand LGBTQ+ terms they don't know!</p>
+      <p>You can request a term here that you want someone else to define.</p>
+      <p> Check out the <a href='http://queerundefined.com/about'>about</a> page to read more about this project.</p>
+    </div>)
+  }
+
+  handleSubmit(e) {
+    if(e) e.preventDefault();
+    const term = this.state.Term;
+    if(term.trim().length === 0) {
+      const snackBarMessage = 'You should provide a term!'
+      this.setState({...this.state,
```

Comment:
`...this.state` is unnecessary because setState merges the provided object with the existing state. you can just provide the fields you want to update